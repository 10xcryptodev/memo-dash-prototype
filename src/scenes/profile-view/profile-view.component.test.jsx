import React from 'react'
import { shallow } from 'enzyme'
import ProfileViewComponent from './profile-view.component'

describe('<ProfileViewComponent />', () => {
  let wrapper

  beforeEach(() => {
    const div = document.createElement('div')
    document.body.appendChild(div)
    const profile = {
      avatarUrl: 'AvatarUrl',
      username: 'Username',
      bio: 'Bio',
      followersCount: 1,
      followingCount: 1
    }
    wrapper = shallow(<ProfileViewComponent profile={profile} />)
  })

  it('renders without crashing', () => {
    expect(wrapper).toMatchSnapshot()
  })
})
