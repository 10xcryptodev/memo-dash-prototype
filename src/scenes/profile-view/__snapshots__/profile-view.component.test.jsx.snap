// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ProfileViewComponent /> renders without crashing 1`] = `
<React.Fragment>
  <Container
    style={
      Object {
        "marginTop": "7em",
      }
    }
  >
    <Grid>
      <GridRow>
        <GridColumn
          width={4}
        >
          <Image
            as="img"
            circular={true}
            size="medium"
            src="AvatarUrl"
            ui={true}
          />
          <Header
            as="h1"
          >
            Username
          </Header>
          <Segment>
            Bio
          </Segment>
        </GridColumn>
        <GridColumn
          width={12}
        >
          <Menu>
            <MenuItem>
              Followers 
              1
            </MenuItem>
            <MenuItem>
              Following 
              1
            </MenuItem>
          </Menu>
        </GridColumn>
      </GridRow>
    </Grid>
  </Container>
</React.Fragment>
`;
